code/modules/clothing/under/jobs/security.dm

```diff
@@ -133,6 +133,13 @@
 	can_adjust = FALSE
 	fitted = FEMALE_UNIFORM_TOP
 
+/obj/item/clothing/under/rank/security/detective/vpndet
+	name = "police nationale patrolman uniform"
+	desc = "Maréchal, nous voilà !"

```

Comment:
is there supposed to be a space before the exclamation mark?

Reply:
Yes